src/main/java/lesson03/conditions/BlackJack.java

```diff
@@ -0,0 +1,42 @@
+package lesson03.conditions;
+/*
+Task 1 (BlackJack)
+Посчитать, сколько очков получает игрок в Блэкджек, если он имеет карты номиналом cardA и cardB.
+Количество очков равняется номиналу большей карты, но не больше чем 21. Если обе карты по номиналу больше 21, то количество очков 0.
+Ограничения: cardA > 0, cardB > 0.
+int blackjackScore(int cardA, int cardB)
+Входные аргументы int cardA, int cardB
+Тип результата int
+ */
+
+import utils.errorhandling.ExceptionHandling;
+
+public class BlackJack {
+    public static void main (String[] args) {
+        System.out.println(blackjackScore(-21,19));
+
+    }
+
+    public static int blackjackScore(int cardA, int cardB){
+        int winningScoreResult = 0;
+        if (cardA > 21 && cardB > 21) {
+            return winningScoreResult;
+        }
+        if (isCardsAreCorrect(cardA, cardB)){
```

Comment:
I suggest using a "validate" method instead "is"  with throwing an exception.

Reply:
Fixed